sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection/edge_detection_with_webcam.cpp

```diff
@@ -25,71 +25,68 @@ int main(int argc, char* argv[]) {
   constexpr auto area = height * width;
 
   cv::VideoCapture cap;
-  // set buffer to 1 frame
-  cap.set(cv::CAP_PROP_BUFFERSIZE, 1);
-  if(!cap.open(0))
+  // If opencv don't find a webcam the program exit here
+  if (!cap.open(0)) {
     return 0;
+  }
 
   // set the size of picture taken by the webcam
   cap.set(cv::CAP_PROP_FRAME_WIDTH, width);
   cap.set(cv::CAP_PROP_FRAME_HEIGHT, height);
 
   selector_defines::CompiledForDeviceSelector selector;
-  queue q {selector, property::queue::enable_profiling()};
+  queue q { selector, property::queue::enable_profiling {} };
 
-  // This infinite loop take a picture from the camera
-  // apply the edge detection filter with the FPGA kernel
-  // display the input and the output in separated windows
-  for(;;){
+  // This infinite loop takes a picture from the camera
+  // applies the edge detection filter with the FPGA kernel
+  // and displays the input and the output in separated windows
+  for (;;) {
     cv::Mat inputColor;
-    // take the picture frim camera
+    // take the picture from the camera
     cap >> inputColor;
 
     cv::imshow("inputColor", inputColor);
     cv::Mat inputRaw, input;
-    cv::cvtColor(inputColor, inputRaw, CV_BGR2GRAY);
     // convert the colored picture into grey
+    cv::cvtColor(inputColor, inputRaw, CV_BGR2GRAY);
     inputRaw.convertTo(input, CV_8UC1);
 
     buffer<uchar> ib{input.begin<uchar>(), input.end<uchar>()};
     buffer<uchar> ob{range<1>{area}};
 
     // The loop break when pressing the Esc key
-    if( cv::waitKey(10) == 27 ) break;
-
-    cv::imshow("input", input);
```

Comment:
Is this on purpose you removed this image?

Reply:
Yes it is redundant with the same instruction at line 101
This was a previous error